packages/core/app/mirage/routes/delivery-rules.js

```diff
@@ -3,7 +3,7 @@ import Mirage from 'ember-cli-mirage';
 
 const TIMESTAMP_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
-export default function() {
+export default function(deliveryType) {
```

Comment:
Can we get get this from `deliveryRule.deliveryType`?

Reply:
updated